### PR TITLE
Add color mapping for cycle segregation using palette_npt

### PR DESCRIPTION
**Pull request by wangzhao0217**

### CODE REVIEW

File: R/osmactive.R [hunk 1]
```diff
@@ -207,7 +207,7 @@ classify_cycle_infrastructure = function(
     stop("Classification type not supported yet. Please open an issue at github.com/nptscot/osmactive.")
   }
 }
-classify_cycle_infrastructure_scotland = function(osm, min_distance = 10) {
+classify_cycle_infrastructure_scotland <- function(osm, min_distance = 10) {
```
Comment (Robinlovelace) [inline R/osmactive.R:210]:
```suggestion
classify_cycle_infrastructure_scotland = function(osm, min_distance = 10) {
```

File: R/osmactive.R [hunk 2]
```diff
@@ -218,38 +218,38 @@ classify_cycle_infrastructure_scotland = function(osm, min_distance = 10) {
       # these by default are not shared with traffic:
       segregated == "yes" ~ "Stepped or footway",
       segregated == "no" ~ "Stepped or footway",
-      TRUE ~ "Mixed traffic"
+      TRUE ~ "Mixed Traffic Street"
```
Comment (Robinlovelace) [inline R/osmactive.R:221]: 👍

File: R/osmactive.R [hunk 3]
```diff
@@ -218,38 +218,38 @@ classify_cycle_infrastructure_scotland = function(osm, min_distance = 10) {
...
     )) |>
     # ...including by name
     dplyr::mutate(detailed_segregation = dplyr::case_when(
       # highways named towpaths or paths are assumed to be off-road
       stringr::str_detect(name, "Path|Towpath|Railway|Trail") &
-        detailed_segregation %in% c("Level track", "Stepped or footway") ~ "Remote cycle track",
+        detailed_segregation %in% c("Level track", "Stepped or footway") ~ "Off Road/Detached Cycle Track",
```
Comment (Robinlovelace) [inline R/osmactive.R:227]: 👍

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Minor change requested.

File: R/osmactive.R (hunk 1 above)
Comment (wangzhao0217) [reply R/osmactive.R:210]: good spot, thx Robin

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**